### DISCUSSION TITLE
Aideck : How to burn the classification demo via JTAG

### DISCUSSION THREAD

**Discussion by eladpar-rnd** [bitcraze/discussions#209] [Q&A] [answered]
Currently the classification README instructs how to burn the img file into the AI-deck using the new method of CFLOADER via radio.
as we ran into problems with this method (mainly the cfloader gets stuck
![image](https://user-images.githubusercontent.com/72129770/185788943-66cd29d3-f22c-487c-84f9-5be7631c97d0.png)
)

we would love to have an example or instructions how to use the Old way burning programs via the JTAG programmer
Any ideas?

**Comment by knmcguire** [comment, marked as answer]
Have you followed the aideck getting started tutorial first? https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/. There are some things to do with the aideck before over air flashing works. 

If you really want to use a programmer, you can use the [flashing instructions](https://www.bitcraze.io/documentation/repository/aideck-gap8-examples/master/getting-started/flashing/) here. Just mind that you will be flashing over the bootloader of the gap8 and over air flashing will not work at all then, but [that can be restored with the instruction in the tutorial](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/#gap8-bootloader-for-older-aidecks-revisions-a-c-)
